react-app/src/app/screens/Game/components/Info/index.js

```diff
@@ -6,18 +6,19 @@ import { GO_TO_MOVE, GO_TO_GAME_START, NEXT_PLAYER } from '../../constants';
 import Button from './components/Button';
 import Info from './layout';
 
-function InfoContainer({ history, status, onClick }) {
+function InfoContainer({ history, status, goToTurn }) {
   const movesList = history.map((step, move) => {
     const text = move ? GO_TO_MOVE + move : GO_TO_GAME_START;
-    return <Button key={move} onClick={onClick} move={move} text={text} />;
+    // eslint-disable-next-line react/no-array-index-key
```

Comment:
Why are you adding this?

Reply:
To disable eslint errors on those lines. Shouldn't I do it ?